### PR TITLE
Next.js 9.5

### LINKED ISSUE #88: Compatibility with nextjs canary
**Is your feature request related to a problem? Please describe.**
Trying to upgrade an app using next-transpile-module@3.3.0 to use nextjs@9.4.5-canary.20 breaks `next dev` with the following error : 
```
TypeError: Cannot read property 'concat' of undefined
    at Object.webpack (/home/jean/dev/node_modules/next-transpile-modules/src/next-transpile-modules.js:102:73)
    at Object.webpack (/home/jean/dev/node_modules/@zeit/next-source-maps/index.js:34:29)
    at getBaseWebpackConfig (/home/jean/dev/node_modules/next/dist/build/webpack-config.js:115:372)
    at async Promise.all (index 1)
    at async HotReloader.start (/home/jean/dev/node_modules/next/dist/server/hot-reloader.js:14:2147)
    at async DevServer.prepare (/home/jean/dev/node_modules/next/dist/server/next-dev-server.js:11:378)
    at async /home/jean/dev/node_modules/next/dist/cli/next-dev.js:21:359
``` 

running ` NODE_OPTIONS='--inspect-brk' next dev     ` allowed me to the internal changes in cssloader config: 
```
> console.log(nextCssLoader.issuer)
undefined
v Object {and: Array(1), not: Array(1)}
 > and:Array(1) ["/home/jean/dev/www"]
 > not:Array(1) [/node_modules/]
```
**Describe the solution you'd like**

I would like the app to start normally :)

### PR DESCRIPTION
**Pull request by martpie**
closes #88 

👀

### CODE REVIEW

File: src/__tests__/__apps__/basic/package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+    "start": "cross-env NODE_PRESERVE_SYMLINKS=1 next start"
   },
-  "author": "",
-  "license": "ISC",
+  "author": "Pierre de la Martinière <pierre.de.la.martiniere@gmail.com>",
+  "license": "MIT",
   "dependencies": {
     "lodash-es": "^4.17.15",
-    "next": "9.3.0",
+    "next": "9.4.5-canary.45",
```
Comment (reckter) [inline src/__tests__/__apps__/basic/package.json:15]:
```suggestion
    "next": "9.5.0",
```
This should probably be 9.5 :)

File: src/__tests__/__apps__/webpack-5/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "scripts": {
+    "dev": "cross-env NODE_PRESERVE_SYMLINKS=1 next",
+    "build": "cross-env NODE_PRESERVE_SYMLINKS=1 next build",
+    "start": "cross-env NODE_PRESERVE_SYMLINKS=1 next start"
+  },
+  "author": "Pierre de la Martinière <pierre.de.la.martiniere@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "lodash-es": "^4.17.15",
+    "next": "9.4.5-canary.45",
```
Comment (reckter) [inline src/__tests__/__apps__/webpack-5/package.json:15]:
```suggestion
    "next": "9.5.0",
```

**Comment by arzafran** [comment]
This should be safe to merge now, right?

File: src/__tests__/__apps__/basic/package.json (hunk 1 above)
Comment (martpie) [reply src/__tests__/__apps__/basic/package.json:15]: yes, it was not released at the time

**Comment by martpie** [comment]
now yes

**Comment by Zertz** [comment]
Merci beaucoup @martpie for the very fast compatibility update ❤️ 🇨🇦

**Pull request merged**